README.md

```diff
@@ -16,6 +16,47 @@ Companies building on OpenPubkey include:
 
 OpenPubkey is a Linux Foundation project. It is open source and licensed under the Apache 2.0 license. This project presently provides an OpenPubkey client and verifier for creating and verifying PK Tokens from Google’s OP (for users) and GitHub’s OP (for workloads).
 
+## Getting Started
+To get started let's walk through a simple message signing example. For conciseness we omit the error handling code, the full running code for this example can be found in [./examples/simple/example.go](./examples/simple/example.go).
+
+
+We start by configuring the OP (OpenID Provider) our client and verifier will use (Google).
+
+```golang
+op := &providers.GoogleOp{
+  ClientID:     "992028499768-ce9juclb3vvckh23r83fjkmvf1lvjq18.apps.googleusercontent.com",
+  ClientSecret: "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", // The client secret is a public value
+  Scopes:       []string{"openid profile email"},
+  RedirURIPort: "3000",
+  CallbackPath: "/login-callback",
+  RedirectURI:  "http://localhost:3000/login-callback",
+}
+```
+
+Next we create the OpenPubkey client and call `opkClient.Auth`:
+```golang
+opkClient, err := client.New(op)
+pkt, err := opkClient.Auth(context.TODO())
```

Comment:
Should some comment be made about `context.TODO()`?

Reply:
What do you suggest?

Reply:
What is it and why is it called `TODO`? What is to be done?

Reply:
Changed this to use `context.Background()` instead. I want to get developers up and running with requiring that make as few choices as possible. `context.TODO()` implies that should replace the context with something else. Instead I'd rather give them a sane default.

Reply:
Should be fixed